src/Pages/ViewList.js

```diff
@@ -1,28 +1,38 @@
 import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
+import EmptyListPrompt from '../components/EmptyListPrompt';
 import { db } from '../lib/firebase';
 
 const ViewList = ({ token }) => {
-  // if (token === null) token = 'default';
-
-  const [list, loading, error] = useCollection(db.collection(token));
-
   return (
     <div>
       <h2>Grocery List</h2>
-
-      {error && <strong>Error: {JSON.stringify(error)}</strong>}
-      {loading && <span>Collection: Loading...</span>}
-      {list && (
-        <ul>
-          <b>Collection: </b>
-          {list.docs.map((doc) => (
-            <li key={doc.id}>{JSON.stringify(doc.data().item)} </li>
-          ))}
-        </ul>
-      )}
+      <List token={token} />
     </div>
   );
 };
 
+const List = ({ token }) => {
+  const [list, loading, error] = useCollection(db.collection(token));
+
+  if (!loading && list.docs.length === 0) {
```

Comment:
This is what I tried to explain @JenniferToops :D

Reply:
The best thing we could do here is turn the conditions on line 23, 24, and 25 into `elseif`s